package.json

```diff
@@ -34,6 +34,7 @@
         "coingecko-api-v3": "0.0.11",
         "cors": "^2.8.5",
         "crc": "^3.8.0",
+        "cross-env": "^7.0.3",
```

Comment:
Isn't it good to add to devDependencies?
npm install --save-dev cross-env

Reply:
npm start is also affected.